instance.go

```diff
@@ -167,7 +170,22 @@ func (inst *TypedInstance[T]) ValueP() T {
 	return t
 }
 
-// Translate transforms the provided [Instance] to an Instance of a different
+// Translate is [Instance.TranslateErr], but panics if an error is encountered.
+//
+// Errors can only occur in Translate if lenses are not written correctly.
+// Eventually, it is hoped that all possible error cases on lenses will be
+// determinable statically, and turned into lineage validity errors that are
+// caught by [BindLineage]. If that goal is reached, this function will never
+// panic, and [Instance.TranslateErr] will be deprecated.
+func (i *Instance) Translate(to SyntacticVersion) (*Instance, TranslationLacunas) {
+	inst, lac, err := i.TranslateErr(to)
+	if err != nil {
+		panic(err)
```

Comment:
Could we remove this panic and return an error instead?

Reply:
that's what `TranslateErr` is for - though maybe it would be better to have a `TranslateP` that panics and `TranslateErr` returns an error

Reply:
I like this idea because of the fact of leaving the simple version of `Translate` open for future releases, optionally moving towards deprecations instead breaking changes - tho it's of course not critical because of the fact we're on `v0.x` 😄